index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
+    <h1> Jaye Lin Resume </h1>
+    <div class="contact">
+      <p> <a href="Illusxia. carbonmade.com"> Portfolio </a><br>
+
+        <a href="Xiaoxxia23@Gmail.com"> Email </a> <br>
+        Baltimore, MD
+      </p>
+    </div>
+
+    <div class="edu">
+      <h3> Education And Training: </h3>
```

Comment:
You have an `h1` and an `h3`, but no `h2`; see the HTML best practices re: [heading elements](https://mica-web.github.io/learn/html/best-practices#heading-tags-are-used-semantically-not-simply-for-presentation).